ios/ReactViewPagerManager.m

```diff
@@ -7,24 +6,23 @@ @implementation ReactViewPagerManager
 
 RCT_EXPORT_MODULE(RNCViewPager)
 
-RCT_EXPORT_VIEW_PROPERTY(initialPage, NSInteger)
```

Comment:
For parity of implementation with Android; implement in js for both.

I suppose it might make more sense to instead implement on the native side for both.